.github/workflows/pythonpackage.yml

```diff
@@ -3,6 +3,31 @@ name: tests
 on: [push, pull_request]
 
 jobs:
+  lint:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Install dependencies
+        uses: actions/setup-python@v2
+        with:
+          python-version: "3.10"
+      - name: Install dependencies
+        run: |
+          pip install --upgrade pip
+          pip install -e .
```

Comment:
i don't think we actually need to install dedupe.

Reply:
Indeed, fixed